pkg/selfupdate/updater.go

```diff
@@ -46,8 +42,7 @@ func New(c Config) (*Updater, error) {
 	var err error
 
 	u := &Updater{
-		githubToken: c.GithubToken,
-		cacheDir:    c.CacheDir,
+		cacheDir: c.CacheDir,
```

Comment:
Removing the `githubToken` config option, as the kubectl-gs repository is public.